packages/story-editor/src/app/story/storyTriggers/storyEvents/onInitialElementAddedRegister.js

```diff
@@ -46,6 +47,18 @@ function OnInitialElementAddedRegister({ currentStory, dispatchStoryEvent }) {
   return null;
 }
 
-OnInitialElementAddedRegister.propType = registerPropTypes;
+OnInitialElementAddedRegister.propTypes = {
+  currentStory: PropTypes.shape({
+    story: PropTypes.shape({
+      pages: PropTypes.arrayOf(StoryPropTypes.page),
```

Comment:
i think Max's idea of checking for elements array here would still be nice: https://github.com/GoogleForCreators/web-stories-wp/pull/11246#discussion_r847784158

Reply:
Oh yeah `StoryPropTypes.page` has it. 
```
StoryPropTypes.page = PropTypes.shape({
  id: PropTypes.string.isRequired,
  animations: PropTypes.arrayOf(PropTypes.shape(AnimationProps)),
  elements: PropTypes.arrayOf(PropTypes.shape(StoryPropTypes.element)),
  overlay: PropTypes.oneOf(Object.values(OverlayType)),
  backgroundAudio: PropTypes.shape({
    resource: BackgroundAudioPropType,
    loop: PropTypes.bool,
    tracks: PropTypes.arrayOf(ResourcePropTypes.trackResource),
  }),
});
```

But since we only need `elements` I'll do that.